app/components/StudioFooter.vue

```diff
@@ -125,6 +125,18 @@
   }
 }
 
+.warning {
+  color: var(--warning);
+}
+
+.info {
+  color: #e3973e;
```

Comment:
is there a reason to use a hex value here instead of one of our existing variable colors?

Reply:
fixed